app/controllers/articles_controller.rb

```diff
@@ -1,18 +1,18 @@
 class ArticlesController < ApplicationController
-  before_action :find_articles, only: [:show, :edit, :update, :destroy]
   def index
     @articles = Article.all
   end
 
   def show
-
+    @article = Article.find(params[:id])
   end
 
   def new
     @article = Article.new
   end
 
   def edit
+    @article = Article.find(params[:id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method